docs/ContribOperators.md

```diff
@@ -451,6 +451,8 @@ This version of the operator has been available since version 1 of the 'com.micr
 <dd>Mask of vocabulary for first step. Words that masked with 0 are not allowed to be generated, and 1 is allowed. Shape is (batch_size, vocab_size)</dd>
 <dt><tt>attention_mask</tt> (optional) : I</dt>
 <dd>Custom attention mask. Shape is (batch_size, sequence_length)</dd>
+<dt><tt>timestamp_enable</tt> (optional) : tensor(bool)</dt>
```

Comment:
How about enable_timestamp

Reply:
If we put timestamp_enable in attributes, users will need to regenerate model if they want to switch timestamp on and off.